packages/framework-provider-aws/src/library/graphql-adapter.ts

```diff
@@ -22,7 +19,7 @@ export async function rawGraphQLRequestToEnvelope(
       requestID,
       eventType,
       connectionID,
-      currentUser: await userEnvelopeFromAuthToken(userPool, request.headers?.Authorization),
+      token: request.headers?.Authorization,
```

Comment:
What if `Authorization` is undefined?

Reply:
@NickSeagull we might have open read models or something like that, so in that case the Authorization is null, then in the graphql message handler we won't check the token.